var/spack/repos/builtin/packages/quicksilver/package.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Quicksilver(MakefilePackage):
+    """Quicksilver is a proxy application that represents some elements of
+    the Mercury workload by solving a simpli?ed dynamic monte carlo
+    particle transport problem.  Quicksilver attempts to replicate the
+    memory access patterns, communication patterns, and the branching or
+    divergence of Mercury for problems using multigroup cross sections."""
+
+    homepage = "https://codesign.llnl.gov/quicksilver.php"
+    git      = "https://github.com/LLNL/Quicksilver.git"
```

Comment:
According to Quicksilver's Github repository:

> Instructions to build Quicksilver can be found in the Makefile. Quicksilver is a relatively easy to build code with no external dependencies (except MPI and OpenMP)

Should we add MPI as a dependency then? Maybe conditional on a variant?